pom.xml

```diff
@@ -500,7 +500,7 @@
         <version.org.wildfly.http-client>1.1.8.Final</version.org.wildfly.http-client>
         <version.org.wildfly.naming-client>1.0.14.Final</version.org.wildfly.naming-client>
         <version.org.wildfly.security.elytron>1.17.0.Final</version.org.wildfly.security.elytron>
-        <version.org.wildfly.transaction.client>1.1.14.Final</version.org.wildfly.transaction.client>
+        <version.org.wildfly.transaction.client>2.0.0.Final</version.org.wildfly.transaction.client>
```

Comment:
This can be removed now; it's only needed if you're overriding the value from the root pom.